### ISSUE TITLE
`set-output` command is deprecated and will be disabled soon

### ISSUE THREAD

**Issue by guwirth** [andreacomo/maven-gav-extractor-github-action#2]
Hi,

using the action `maven-gav-extractor` I get the warning below. Are you planning to fix this in a new release?
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Regards,

```
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

Sample:
```
Run andreacomo/maven-gav-extractor@v2
  with:
    pom-location: /home/runner/work/sonar-cxx/sonar-cxx/pom.xml
  env:
    JAVA_HOME: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/11.0.16-1/x64
    JAVA_HOME_11_X64: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/11.0.16-1/x64
Run VALUE=$(mvn -f /home/runner/work/sonar-cxx/sonar-cxx/pom.xml $EVALUATE -Dexpression=project.groupId -q -DforceStdout)
  VALUE=$(mvn -f /home/runner/work/sonar-cxx/sonar-cxx/pom.xml $EVALUATE -Dexpression=project.groupId -q -DforceStdout)
  echo "::set-output name=group-id::$VALUE"
  
  VALUE=$(mvn -f /home/runner/work/sonar-cxx/sonar-cxx/pom.xml $EVALUATE -Dexpression=project.artifactId -q -DforceStdout)
  echo "::set-output name=artifact-id::$VALUE"
  
  VALUE=$(mvn -f /home/runner/work/sonar-cxx/sonar-cxx/pom.xml $EVALUATE -Dexpression=project.version -q -DforceStdout)
  echo "::set-output name=version::$VALUE"
  
  VALUE=$(mvn -f /home/runner/work/sonar-cxx/sonar-cxx/pom.xml $EVALUATE -Dexpression=project.name -q -DforceStdout)
  echo "::set-output name=name::$VALUE"
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    JAVA_HOME: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/11.0.16-1/x64
    JAVA_HOME_11_X64: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/11.0.16-1/x64
    EVALUATE: org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

**Comment by andreacomo** [comment]
Thanks for pointing it out. I will fix asap in a new release

**Comment by andreacomo** [comment]
Fixed in commit https://github.com/andreacomo/maven-gav-extractor/commit/0112b77b504856b76747926a573b1b0545b53824

Shifted `v2` tag to point to the fixed release version `v2.0.1`

**Comment by guwirth** [comment]
Hi @andreacomo, I did a test it's working! Thx
